### ISSUE TITLE
Node name problem

### ISSUE THREAD

**Issue by water110** [kijai/ComfyUI-Florence2#73]
![image](https://github.com/user-attachments/assets/1ccd1f43-b9a9-4ef8-b46c-ffb2650dc053)
I installed these two plugins:kijai/ComfyUI-Florence2、kijai/ComfyUI-segment-anything-2，I used some nodes of these two plug-ins, but the name of the plug-in displayed by the node is comfyui-tensorop, as shown in the figure above, which is very strange to me.
I have installed this plug-in comfyui-tensorop before, but later I saw that you mentioned a name conflict ([https://github.com/un-seen/comfyui-tensorops/issues/1](https://github.com/un-seen/comfyui-tensorops/issues/1)), and that this plugin functions in the same way as the kijai/ComfyUI-Florence2 one, so I uninstalled it. so I uninstalled comfyui-tensorop, but the above problem still occurs. Please help troubleshoot. @kijai

**Comment by kijai** [comment]
It is because they copied my nodes and ComfyUI-Manager, which shows those tags, just matches it to that for whatever reason in it's database. It's highly annoying, but shouldn't affect functionality.

**Comment by kijai** [comment]
> Yes, it really doesn't affect the actual functionality, it just looks awkward.

Indeed, I've sent them issue report as you noticed, I've also complained about it in official Comfy discord, I've been told these kind of issues will be resolved in the future.

**Comment by kijai** [comment]
Raised an issue in the Manager repo now, if you'd like to add to that:

https://github.com/ltdrdata/ComfyUI-Manager/issues/1065
